### PR TITLE
NetCore CsProj conversion tool

### PR DESCRIPTION
**Pull request by MiYanni**
## Description
Created a tool to do conversion from the netframework csproj files to the netcore csproj files.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#making-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by MiYanni** [comment]
@azuresdkci Retest this please

**Comment by vladimir-shcherbakov** [comment]
@MiYanni what is the status of the PR? Are we waiting for the travis-ci to pass? What about the conflict?

**Comment by MiYanni** [comment]
@vladimir-shcherbakov It depends on if we want to merge it or not (entirely). Right now, it can be used as a reusable tool, but it isn't in a state to be part of our analysis build process.

**Comment by MiYanni** [comment]
Updated tool to be used as a validation step after //build. Accepts ResourceManager path and does validation by default.

File: .gitignore [hunk 1]
```diff
@@ -233,3 +233,4 @@ Package
 /src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/bin
 /src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Providers/bin
 /src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.ServiceClientAdapter/bin
+/tools/NetCoreCsProjSync/NetCoreCsProjSync/Properties/launchSettings.json
```
Comment (markcowl) [inline .gitignore]: do we want to have a single rule for launchsettings.json?

File: tools/NetCoreCsProjSync/NetCoreCsProjSync/NetCoreCsProjGenerator.cs [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+        {
+            "Aks",
+            "AnalysisServices.Dataplane",
+            "Compute.ManagedService",
+            "Profile",
+            "RecoveryServices.Backup.Logger",
+            "Tags"
+        };
+
+        public static readonly Dictionary<string, List<string>> ModuleMap = new Dictionary<string, List<string>>
```
Comment (markcowl) [inline tools/NetCoreCsProjSync/NetCoreCsProjSync/NetCoreCsProjGenerator.cs]: Can we make this data external, so we don't have to edit the code to add a new module or remove a module from the skip list?

File: .gitignore (hunk 1 above)
Comment (MiYanni) [reply .gitignore]: Fixed.

File: tools/NetCoreCsProjSync/NetCoreCsProjSync/NetCoreCsProjGenerator.cs (hunk 2 above)
Comment (MiYanni) [reply tools/NetCoreCsProjSync/NetCoreCsProjSync/NetCoreCsProjGenerator.cs]: Fixed. Allowed both `ModuleMap` and `ModuleSkipList` to be loaded from .json files.

**Comment by markcowl** [comment]
on demand run here: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/758/

**Pull request merged**